File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.c
int finalValueAfterOperations(char** operations, int operationsSize) {
    int x=0;
        for(int i=0;i<operationsSize;i++){
            if(strcmp(operations[i],"++X")==0) ++x;
            if(strcmp(operations[i],"X++")==0) x++;
            if(strcmp(operations[i],"--X")==0) --x;
            if(strcmp(operations[i],"X--")==0) x--;
        }
        return x;
}

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        long x=0;
        for(String sr: operations){
            if(sr.equals("++X")) ++x;
            if(sr.equals("X++")) x++;
            if(sr.equals("X--")) x--;
            if(sr.equals("--X")) --x;
        }
        return (int) x;
    }
}